styles/components/_search.scss

```
@@ -91,6 +91,24 @@
 .search-bar-container {
   position: relative;
 
+  .search-bar-collection-search {
+    z-index: 2;
+    position: absolute;
+    left: 41px;
+    top: 9px;
+    height: 23px;
```

can we use rem instead?
```suggestion
    left: 2.5rem;
    top: 0.5rem;
    height: 1.5rem;
```